src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -85,6 +86,29 @@ air_pressure(ts::ThermodynamicState) =
   air_pressure(air_temperature(ts), air_density(ts), PhasePartition(ts))
 air_pressure(ts::PhaseDry) = air_density(ts) * gas_constant_air(ts) * air_temperature(ts)
 
+"""
+    linearized_air_pressure(ρ, ρe_tot, Φ[, q::PhasePartition])
+
+The air pressure, linearized around the reference state, from the equation of state
```

Comment:
'linearized around a dry rest state' would be better--the linearization does not actually depend on what the reference state is, as long as it is dry (q=0) and at rest (u=0).

Reply:
Good point, I'll update this.